src/logic/safe/transactions/gas.ts

```diff
@@ -29,65 +24,67 @@ const estimateDataGasCosts = (data: string): number => {
   return data.match(/.{2}/g)?.reduce(reducer, 0)
 }
 
-export const estimateTxGasCosts = async (
+// https://docs.gnosis.io/safe/docs/docs5/#pre-validated-signatures
+export const getPreValidatedSignatures = (from: string): string => {
+  return `0x000000000000000000000000${from.replace(
+    EMPTY_DATA,
+    '',
+  )}000000000000000000000000000000000000000000000000000000000000000001`
+}
+
+// @todo (agustin) check if this can be replaced by shouldExecuteTransaction() and refactor
```

Comment:
I think this comment can be removed

Reply:
I added this to take that in make for a future refactor of the gas functions

Reply:
a possibility could be status: loading | success | error

Reply:
? didn't get that

Reply:
sorry, it was for the above comment :grin: